src/test/java/io/debezium/connector/yugabytedb/common/TestBaseClass.java

```diff
@@ -194,7 +182,12 @@ public void taskStarted() {
 
   protected int consumeAvailableRecords(Consumer<SourceRecord> recordConsumer) {
     List<SourceRecord> records = new LinkedList<>();
-    consumedLines.drainTo(records);
+
```

Comment:
Nit: Do you need to copy the records to another list? Can `consumedLines` be used directly?

Reply:
Fixed